react-hyf/react-hyf-client/src/homework_week3/users.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import fetchUsers from './Components/fetchData';
+import RenderUser from './Components/renderUser';
+
+class Users extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
```

Comment:
perhaps initializing the users property here would make it a better use, in case of you using a strict mode in the future.